tests/test_theseus_report.py

```
@@ -0,0 +1,140 @@
+            FixtureRunner(), FIXTURES / "os-release", uptime_path=Path("/nonexistent/uptime")
+        )
+        states = {item["collector"]: item["state"] for item in report_data["collection_status"]}
+        self.assertEqual(states["uptime"], "unavailable")
+        self.assertNotIn("uptime_seconds", report_data["volatile"])
+
+    def test_schema_rejects_report_missing_a_known_collector_status(self):
+        report_data = report.build_report(FixtureRunner(), FIXTURES / "os-release")
+        report_data["collection_status"] = [
+            item for item in report_data["collection_status"] if item["collector"] != "nvcc"
+        ]
+        with self.assertRaises(report.ReportError):
```

Ruff (0.16.2)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)